packages/editor/src/store/actions.js

```diff
@@ -177,15 +178,26 @@ export const savePost =
 				edits,
 				options
 			);
-		dispatch( { type: 'REQUEST_POST_UPDATE_FINISH', options } );
 
-		const error = registry
+		let error = registry
 			.select( coreStore )
 			.getLastEntitySaveError(
 				'postType',
 				previousRecord.type,
 				previousRecord.id
 			);
+
+		if ( ! error ) {
+			await applyFilters(
+				'editor.SavePost',
```

Comment:
The other question is whether we want this to be stable or internal. Given the questioning about hook vs filter... I might think we should make it unstable first. WDYT?

Reply:
I'm fine with making it unstable and I'm mainly concerned about what are the right parameters for the action. is `editor.__unstableSavePost` a good name? Is there prior art for unstable/experimental actions and filters?

Reply:
> editor.__unstableSavePost a good name

works for me.

> Is there prior art for unstable/experimental actions and filters?

I don't know :) (At least, personally I don't remember any)

> arguments

We can start with none, right?

Reply:
OK, renamed the action to `editor.__unstableSavePost`.

> We can start with none, right?

I'm starting with `options` because I need to read `options.isAutosave` 🙂 This particular action doesn't need anything else, not even basic stuff like the post ID or the post edits.